Tools/clinic/libclinic/parse_args.py

```diff
@@ -447,6 +470,55 @@ def parse_option_groups(self) -> None:
         parser_code = '    {option_group_parsing}'
         self.parser_body(parser_code)
 
+    def _parse_vararg(self) -> str:
+        paramname = self.varpos.converter.parser_name
+        if self.fastcall:
+            if self.limited_capi:
+                if min(self.pos_only, self.min_pos) < self.max_pos:
+                    size = f'Py_MAX(nargs - {self.max_pos}, 0)'
+                else:
+                    size = f'nargs - {self.max_pos}' if self.max_pos else 'nargs'
+                return f"""
+                    {paramname} = PyTuple_New({size});
+                    if (!{paramname}) {{{{
```

Comment:
```suggestion
                    if ({paramname} == NULL) {{{{
```

for consistency with the else branch.

Reply:
Okay, it makes sense. It seems that most of the generated code use `== NULL` and `!= NULL`.

Reply:
Do we really need to generate thousands of lines of generated diffs for these unneeded style changes?